Libraries/LibWeb/DOM/CharacterData.cpp

```diff
@@ -95,8 +95,14 @@ WebIDL::ExceptionOr<void> CharacterData::replace_data(size_t offset, size_t coun
     full_data.extend(inserted_data);
     full_data.append(after_data.data(), after_data.length_in_code_units());
     Utf16View full_view { full_data };
+
+    auto previous_data = m_data;
```

Comment:
We should probably mark this as `// AD-HOC:` since we're deviating from the spec?